### PR TITLE
Fix const in enums

### PR DESCRIPTION
**Pull request by Eviljeks**
Fixes https://github.com/phpactor/phpactor/issues/2540

### CODE REVIEW

File: lib/WorseReferenceFinder/Tests/Unit/WorseReflectionDefinitionLocatorTest.php [hunk 1]
```diff
@@ -322,6 +322,20 @@ public function testLocatesCase(): void
         $this->assertTypeLocation($location->first(), 'FoobarEnum.php', 24, 33);
     }
 
+    public function testLocatesEnumConst(): void
+    {
+        if (!defined('T_ENUM')) {
```
Comment (dantleech) [inline lib/WorseReferenceFinder/Tests/Unit/WorseReflectionDefinitionLocatorTest.php:327]: Can drop this as the minimum version is 8.1 (which includes enums)

**Review by dantleech — COMMENTED** [review]
Very nice, thankyou. Just one small comment.

File: lib/WorseReferenceFinder/Tests/Unit/WorseReflectionDefinitionLocatorTest.php (hunk 1 above)
Comment (Eviljeks) [reply lib/WorseReferenceFinder/Tests/Unit/WorseReflectionDefinitionLocatorTest.php]: I dropped all checks for `defined('T_ENUM')`, is it ok?

**Comment by dantleech** [comment]
yes perfect, thanks!

**Pull request merged**